plugins/woocommerce/tests/php/src/Internal/Admin/ReviewsListTableTest.php

```diff
@@ -11,4 +13,24 @@
  */
 class ReviewsListTableTest extends WC_Unit_Test_Case {
 
+	/**
+	 * Returns a new instance of the ReviewsListTable class.
+	 *
+	 * @return ReviewsListTable
+	 */
+	protected function get_reviews_list_table() : ReviewsListTable {
+		return new ReviewsListTable( [ 'screen' => 'product_page_product-reviews' ] );
+	}
```

Comment:
I made this method because I imagine future tests might also want an instance of the class and it would save us having to make the array each time.